core/fuzzer.cpp

```diff
@@ -159,16 +159,16 @@ std::unique_ptr<p4::v1::WriteRequest> P4RuntimeFuzzer::produceWriteRequest(bool
             Utils::getRandInt(0, 4) == 0) {
             continue;
         }
-        // TODO: Make this a configurable variable and increase.
-        auto maxEntryGenCnt = Utils::getRandInt(1, 5);
+
+        // The maximum number of entries we are trying to generate for a table.
+        auto maxEntryGenCnt = getProgramInfo().getFuzzerConfig().getMaxEntryGenCnt();
         // The maximum attempts we are trying to generate an entry.
-        int attempts = 0;
+        int attempts = getProgramInfo().getFuzzerConfig().getAttempts();
```

Comment:
Do not replace the initialization with this. The maximum attempts is the 100 constant at line 172.